src/common/constants.ts

```diff
@@ -99,7 +105,12 @@ export const onboardingSteps = {
   FINISH: 6,
 };
 
-export const countryCodes = [{ code: '+62' }, { code: '+38' }];
+export const countryCodes = [
+  { code: '+62' },
+  { code: '+38' },
+  { code: '+1' },
+  { code: '' },
+];
```

Comment:
I was the same changas here - https://github.com/ZenBit-Tech/code-lions_fe/pull/26/files. please connect together and check why it happened. I don't think both of you make exact same changes

Reply:
Yes, I used Arsenia's implementation of phone input with select in my personal profile page. But because her part of code was not in develop branch, had to add same constants.